<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workplans</class>
 <widget class="QWidget" name="workplans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 250);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(176, 196, 222);</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>50</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Направление деятельности</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Задачи</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ожидаемый результат</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Срок выполнения до</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ответственный</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>591</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="direction_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tasks_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="result_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="deadlines_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="main_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(YYYY-MM-DD)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>270</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(ФИО)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
